sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'sass-rails'
 
   gem.add_dependency 'hydra-batch-edit', '~> 1.1'
+  gem.add_dependency 'hydra-head', '~> 9.5.0'
```

Comment:
Yes, unfortunately, because of https://github.com/projecthydra/sufia/issues/1530